### ISSUE TITLE
Не работает анимация доставания оружия, при нажатии кнопки выбора ранее использованного оружия (lastinv)

### ISSUE THREAD

**Issue by Vladislav4KZ** [Velaron/cs16-client#27]
У меня баг пока воспроизводится только при переключении между Desert Eagle и M4A1.

**Comment by Vladislav4KZ** [comment]
Также анимация не воспроизводится при подборе M4A1, когда несёшь Deagle

**Comment by Vladislav4KZ** [comment]
И когда бросаешь эмку, у дигла анимация тоже не воспроизводится

**Comment by Vladislav4KZ** [comment]
С другим оружием видимо такое не происходит.

Любой пистолет + M4A1 – есть анимация
Desert Eagle + Любое другое оружие – тоже есть анимация

**Comment by Velaron** [comment]
Не в глушителе надетом ли дело?

**Comment by Vladislav4KZ** [comment]
> Не в глушителе надетом ли дело?

Да, баг воспроизводится при переключении между Desert Eagle и M4A1 с глушителем.
При переключении с других пистолетов на M4A1 с глушителем, анимация всё же воспроизводится.

Даже USP'а с глушителем, это дело не касается.

**Comment by Vladislav4KZ** [comment]
С гранатами кстати то же самое

**Comment by Vladislav4KZ** [comment]
При переключении с гранаты на гранату кнопкой выбора ранее использованного оружия, анимация не воспроизводится.

Баг воспроизводится на всех трёх гранатах.
